src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));
+    }
```

Comment:
avoid static methods as much as possible, because we are using OOP style, not procedural.
also it's required in the task (read carefully: method, not function)
![image](https://github.com/mate-academy/jv-lottery/assets/73041091/dcf9742c-d7f4-4650-a5af-7f5393c4c873)